qa/rpc-tests/cfund-paymentrequest-payout.py

```diff
@@ -169,36 +167,30 @@ def run_test(self):
         for x in range(self.num_nodes):
             assert(self.nodes[x].getbestblockhash() == bestBlockHash)
             assert(self.nodes[x].getpaymentrequest(paymentReq)["state"] == 1)
-            assert(self.nodes[x].getpaymentrequest(paymentReq)["paidOnBlock"] == "0000000000000000000000000000000000000000000000000000000000000000")
+            assert(self.nodes[x].getpaymentrequest(paymentReq)["stateChangedOnBlock"] != "0000000000000000000000000000000000000000000000000000000000000000")
 
         # disconnect the nodes and generate the payout on each node
         url = urllib.parse.urlparse(self.nodes[1].url)
         self.nodes[0].disconnectnode(url.hostname+":"+str(p2p_port(1)))
 
         time.sleep(1)
 
-        while self.nodes[0].getpaymentrequest(paymentReq)["paidOnBlock"] == "0000000000000000000000000000000000000000000000000000000000000000":
+        while self.nodes[0].getpaymentrequest(paymentReq)["state"] == 6:
```

Comment:
isn't state 6 the final state of `paid`? Or will it move from this to something else eventually? The diff says the while condition was previously `paidonblock == "000.."` which i thought meant loop until it has been paid, but now the while runs until the state is no longer paid? I must be misinterpreting this, the test passes.

Reply:
d3e786ff

```
$ qa/rpc-tests/cfund-paymentrequest-payout.py 
Initializing test directory /var/folders/qh/7_l8lmqx2sb5ncxgtzzyb51r0000gn/T/testfn5g4cou/27872
Stopping nodes
Cleaning up
Tests successful
```